pkg/oc/cli/cmd/set/volume.go

```diff
@@ -411,9 +415,6 @@ func (v *VolumeOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, out,
 		}
 		v.AddOpts.ClaimSize = q.String()
 	}
-	if len(v.AddOpts.DefaultMode) == 0 {
```

Comment:
That's actually good piece of code that should be left here. Removing it from here you're creating two places where you're setting the default mode.

Reply:
@soltysh Thank you. But if DefaultMode was set here, other storage type (`emptydir`,`hostpath` ..) will be broken with current validation check. You can [refer to the eror](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/19169/test_pull_request_origin_cmd/12077/)
Is it OK to change them? I think codes under `func (a *AddVolumeOptions) Validate(isAddOp bool)` need a big change.

Reply:
Feel free to fix it as necessary, but ensure you don't create duplicate code paths.